README.md

```diff
@@ -1,6 +1,37 @@
 # Trackr
+
+--------------------------
+
 ![our epic trackr logo](https://i.imgur.com/CqHlNde.png)
 
 Event Trackr Android app. Allows users to track birthdays and anniversaries of friends and family. 
 
+## Configuring the Project
+
+To keep things organized, this project uses the modules feature of IntelliJ. IntelliJ modules separate out the
+phase-specific folders from the source code, and keep the project root from getting cluttered by gradle build files (
+which are only necessary for the source code).
+
+On cloning the project (and opening it in IntelliJ idea), you will have to mark the `trackr` folder as its own module. 
+To do this, go to:
+
+- `File` -> `Project Structure`
+- `Project Settings` -> `Modules` -> `+` (in the top left corner) -> `Import Module`
+- Select the `trackr` folder in this project -> `Okay`
+- `Import Module from externel model` -> `Gradel` -> `Finish`
+
```

Comment:
I tried replicating the steps on a new build up to here but I got the following error:

Unsupported Java. 
Your build is currently configured to use Java 17 and Gradle 7.1.

Possible solution:
 - Use Java 16 as Gradle JVM: Open Gradle settings 
 - Open Gradle wrapper settings, change `distributionUrl` property to use compatible Gradle version and reload the project

The JDKs aren't downloading by themselves.